### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- Docstrings updated
- New tests added
- Minor edit in menu messages

### CODE REVIEW

File: pyrgg/params.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
         1- Pyrgg engine
         """
         )],
-    2: ["file_name", "- File Name (not empty) : "],
+    2: ["file_name", "- File Name (Not Empty) : "],
```
Comment (sadrasabouri) [inline pyrgg/params.py:13]: why?

File: pyrgg/params.py [hunk 2]
```diff
@@ -65,7 +65,7 @@
     7: ["direct", "- Undirected[0] or Directed[1]"],
     8: ["self_loop", "- No Self Loop[0] or Self Loop[1]"],
     9: ["multigraph", "- Simple[0] or Multigraph[1]"],
-    10: ["config", "- Save config[1] or not[0]"],
+    10: ["config", "- Save Config[1] or Not[0]"],
```
Comment (sadrasabouri) [inline pyrgg/params.py:68]: why?

**Review by sadrasabouri — COMMENTED** [review]
Minor questions (comments) are applied.

File: pyrgg/params.py (hunk 1 above)
Comment (sepandhaghighi) [reply pyrgg/params.py:13]: Just for uniformity

File: pyrgg/params.py (hunk 2 above)
Comment (sepandhaghighi) [reply pyrgg/params.py:68]: Just for uniformity

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**